src/01-general/history.apib

```diff
@@ -14,3 +14,25 @@ Retrieve a list of records about an item.
                          + contact
                          + company
                 + id: `6e7fe84d-d4b3-4723-abae-9bc082d8f65a` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `6e7fe84d-d4b3-4723-abae-9bc082d8f65a` (string)
+                + title: `Marc was on a call with John from this company` (string)
+                + description: `An extended description` (string)
+                + recorded_at: `2017-05-09T11:25:11+00:00` (string)
+                + relates_to (object)
```

Comment:
B: 
```suggestion
                + relates_to (array)
```

(Unless it would be a single entity, then the object beneath it doesn't make sense.)

However I'm also not sure about the naming. There is also still an [open proposal](https://github.com/teamleadercrm/api/pull/265) on adding `relates_to` to time tracking and timers, where it represents the related entities of a time tracking/timer's subject. I.e. a project ID that relates to the milestone, and the customer of the project. So, a list of entities that we can deduce later.

Here it's actually kind of an actor, or entity that was acted on, right?